app/src/org/commcare/tasks/ProcessAndSendTask.java

```diff
@@ -466,6 +467,18 @@ protected int getSuccessfulSends() {
         return successes;
     }
 
+    protected String getLabelForFormsSent() {

```

Comment:
Gotcha. If you want to change the static value of the message associated with `sync.success.sent.singular` to be "1 Form sent to the server!", that's definitely fine with me, but it doesn't make sense to be treating it like we're passing a dynamic value. 

Also agree that "No forms sent to the server" is slightly better than saying 0, so feel free to change that